heartbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[running-on-kubernetes]]
+=== Running {beatname_uc} on Kubernetes
+
+{beatname_uc} <<running-on-docker,Docker images>> can be used on Kubernetes to
+check resources uptime.
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {beatname_uc} has not yet been
+released, so no Docker image is currently available for this version.
+
+endif::[]
+
+
+[float]
+==== Kubernetes deploy manifests
+
+A single {beatname_uc} can check for uptime of the whole cluster.
+
+Everything is deployed under `kube-system` namespace, you can change that by
+updating the YAML file.
+
+To get the manifests just run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/kubernetes/{beatname_lc}-kubernetes.yaml
+------------------------------------------------
+
+[WARNING]
+=======================================
+If you are using Kubernetes 1.7 or earlier: {beatname_uc} uses a hostPath volume to persist internal data, it's located
+under /var/lib/{beatname_lc}-data. The manifest uses folder autocreation (`DirectoryOrCreate`), which was introduced in
```

Comment:
It seems that there is no `DirectoryOrCreate` volume in the manifests.

Reply:
Good catch! 0db39e7

Reply:
👍